app/models/job.js

```diff
@@ -200,6 +200,24 @@ export default Model.extend(DurationCalculations, {
     }
   }),
 
+  isTrustySudoFalse: Ember.computed('queue', function () {
+    if (this.get('queue') === 'builds.ec2') {
+      return true;
+    }
+  }),
+
+  isRetiredMacImage: Ember.computed('queue', function () {
+    if (this.get('queue') === 'builds.macstadium6')
+      && (this.get('config.osx_image') === 'beta-xcode6.1' ||
+      this.get('config.osx_image') === 'beta-xcode6.2' ||
+      this.get('config.osx_image') === 'beta-xcode6.3' ||
+      this.get('config.osx_image') === 'xcode7' ||
+      this.get('config.osx_image') === 'xcode7.1' ||
+      this.get('config.osx_image') === 'xcode7.2') {
+      return true;
+    }
+  }),
```

Comment:
Sadly, unlike the above, there‘s no built-in way to perform this match. Just for a little less verbosity, I’d probably write it like this:

```
isRetiredMacImage: Ember.computed('queue', 'config.osx_image', function () {
  const isMacStadium6 = this.get('queue') === 'builds.macstadium6');
  const retiredImages = ['beta-xcode6.1', 'beta-xcode6.2', 'beta-xcode6.3', 'xcode7', 'xcode7.1', 'xcode7.2'];

  return isMacStadium6 && retiredImages.includes(this.get('config.osx_image'));
}
```

This is much more opinion/style territory than the `equals` computed property, as I just get tired of giant if\ors.

Note that I added the `config.osx_image` dependent property key in the function definition because if the value happens to change (maybe it starts off empty and is filled in after an Ajax call?), the computed property needs to recalculate.